easyconfigs/d/DualSPHysics/DualSPHysics-5.0.233-foss-2022a.eb

```
@@ -0,0 +1,50 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+import os as _os
+
+easyblock = 'MakeCp'
+
+name = 'DualSPHysics'
+version = '5.0.233'
+
+homepage = "https://dual.sphysics.org"
+description = """DualSPHysics is based on the Smoothed Particle Hydrodynamics model named SPHysics (www.sphysics.org).
+ The code is developed (GNU Lesser General Public License) to study free-surface flow phenomena where Eulerian methods
+ can be difficult to apply. DualSPHysics is a set of C++, CUDA and Java codes designed to deal with real-life
+ engineering problems."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/DualSPHysics/DualSPHysics/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s_makefiles.patch']
+checksums = [
+    {'v5.0.233.tar.gz': '660a67dc8deed7a38f63e6f462071cbb980e268296f12a5c182aad85b4c0e9e9'},
+    {'DualSPHysics_makefiles.patch': '2418ef17fa0c47e41be523e81588cc92401ac5554f48fbf900e567438c721e79'},
+]
+
+dependencies = [
+    ('Java', '11', '', True),
+]
+
+# parallelise first make command as well
+_ntasks = _os.getenv('SLURM_NTASKS')
+build_cmd = 'cd src/source && make -f Makefile_cpu -j %s && ' % _ntasks
```

```suggestion
build_cmd = 'cd src/source && make -f Makefile_cpu -j %(parallel)s && '
```

That uses the inbuild parallel value set by EB and removes the need to do the `import os as _os`.